pages/sdk-onramp-kit/reference/monerium-pack.md

```diff
@@ -39,9 +40,11 @@ MoneriumConfig {
 
 The `clientId` is the secret representing the "Authorization Code Flow" for your Monerium account. To get your `clientId`:
 
-1. Log in to [your account](https://monerium.dev) 
+1. Log in to [your account](https://monerium.dev).
 2. Create a [new application](https://monerium.dev/docs/getting-started/create-app).
 
+The `redirectUrl` is an URI that will be used to send the user back to the app after they complete the authentication process.
```

Comment:
Should this property be added to the `MoneriumConfig` type in line 35?

Reply:
yeah, it is there but was not updated @dasanra

Reply:
And MoneriumConfig is MoneriumProviderConfig 😬